Riot/Modules/SetPinCode/SetPinCoordinatorBridgePresenter.swift

```diff
@@ -115,11 +123,14 @@ final class SetPinCoordinatorBridgePresenter: NSObject {
         }
     }
     
-    func dismiss() {
+    func dismissWithMainAppWindow(_ window: UIWindow) {
         guard let coordinator = self.coordinator else {
             return
         }
         coordinator.toPresentable().view.removeFromSuperview()
```

Comment:
```suggestion
```

Perhaps this is not useful anymore then (we could maybe set the pinCoordinator to nil though - I would expect the view to die for sure if we remove both its window and its coordinator)